packages/commercetools/theme/pages/Checkout/Shipping.vue

```diff
@@ -113,14 +140,22 @@
           />
           </ValidationProvider>
         </div>
+        <SfButton
+          v-if="!provideAddress"
+          class="form__action-button form__action-button--margin-bottom"
+          type="submit"
+          @click.native="provideAddress = true"
+        >
+          Add new address
+        </SfButton>
         <SfHeading
-          v-if="isShippingAddressCompleted && !dirty"
+          v-if="isShippingAddressCompleted && !dirty && !justChangedAddress"
```

Comment:
`isShippingAddressCompleted && !dirty && !justChangedAddress` is repeats few times, so it should be extracted to computed property

Reply:
Unfortunately, `dirt` is scoped slot so I could only create method here:
```js
const canContinueToPayment = dirty => isShippingAddressCompleted.value && !dirty && !justChangedAddress;
```